### PR TITLE
Update year, wording tweak

### PR DESCRIPTION
**Pull request by richsalz**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -2,15 +2,15 @@
   LICENSE ISSUES
   ==============
 
-  The OpenSSL toolkit stays under a dual license, i.e. both the conditions of
+  The OpenSSL toolkit stays under a double license, i.e. both the conditions of
```
Comment (levitte) [inline LICENSE]: Not sure I agree.  "dual license" is a common terminology that's been used for years and years

Comment (richsalz) [inline LICENSE]: Right, but we are using it wrongly.  Dual as usually used means "a or b"  Ours is double, meaning "a and b"

Comment (levitte) [inline LICENSE]: Ah!  Subtle...

**Comment by richsalz** [comment]
thanks.

**Pull request closed without merging**